firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/SignInResultActivity.java

```diff
@@ -25,6 +25,8 @@
 public class SignInResultActivity extends AppCompatActivity {
```

Comment:
Does this activity need to be public?

Reply:
I believe it does because it is in the manifest. I tried making it package private and the app would close on start up without any error,  but changing it back to public fixes that issue. 

 https://github.com/firebase/firebase-android-sdk/blob/dd1c33608f3dfa143317b5652659989848742b47/firebase-app-distribution/src/main/AndroidManifest.xml#L37

Reply:
Ah! That makes sense. Thanks for the clarification.